pkg/app/init.go

```diff
@@ -0,0 +1,135 @@
+package app
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"go.uber.org/zap"
+	"helm.sh/helm/v3/pkg/cli"
+
+	"github.com/helmfile/helmfile/pkg/helmexec"
+)
+
+const (
+	HelmInstallCommand      = "curl -fsSL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash"
+	HelmRequiredVersion     = "v2.10.0"
+	HelmDiffRequiredVersion = "v3.4.0"
+)
+
+type helmRequiredPlugin struct {
+	name    string
+	version string
+	repo    string
+}
+
+type HelmfileInit struct {
+	helmBinary string
+	logger     *zap.SugaredLogger
+	runner     helmexec.Runner
+}
+
+func NewHelmfileInit(helmBinary string, logger *zap.SugaredLogger, runner helmexec.Runner) *HelmfileInit {
+	return &HelmfileInit{helmBinary: helmBinary, logger: logger, runner: runner}
+}
+func (h *HelmfileInit) UpdateHelm() error {
+	return h.InstallHelm()
+}
+
+func (h *HelmfileInit) InstallHelm() (err error) {
+	if runtime.GOOS == "windows" {
+		return fmt.Errorf("windows platform, please install helm manually, installation steps: https://helm.sh/docs/intro/install/")
```

Comment:
I would prefer a check for choco or scoop in order to show either `scoop install helm` or `choco install kubernetes-helm`, and even `busybox`, otherwise this message

Reply:
great idea, i'll finish it.